src/modules/LanItems.rb

```diff
@@ -2667,16 +2665,12 @@ def export_udevs(devices)
           )
         end
         Builtins.foreach(devs) do |device|
-          driver = Convert.convert(
-            SCR.Execute(
-              path(".target.bash_output"),
-              Builtins.sformat(
-                "driver=$(ls -l /sys/class/net/%1/device/driver);echo ${driver##*/}|tr -d '\n'",
-                device
-              )
-            ),
-            from: "any",
-            to:   "map <string, any>"
+          driver = SCR.Execute(
+            path(".target.bash_output"),
+            Builtins.sformat(
+              "driver=$(/usr/bin/ls -l /sys/class/net/%1/device/driver); /usr/bin/echo ${driver##*/} | /usr/bin/tr -d '\n'",
```

Comment:
i think that ```echo``` and ```/usr/bin/echo``` are different things ... and I don't know if the later one supports bashisms like ```${...##*/}```

Reply:
well bad example, expansion is done by shell ... but anyway. Is there any known difference between ```echo``` and ```/usr/bin/echo```?

Reply:
I am not much aware of it.